.travis.yml

```diff
@@ -1,11 +1,22 @@
 language: java
 
-# only java 8 supported for now
-jdk: oraclejdk8
+# test with java 9 support
+jdk: 
+- openjdk11
+- openjdk12
+- openjdk13
+
+# using the right gradle version
+before_install: 
+- wget https://services.gradle.org/distributions/gradle-6.2.1-bin.zip 
+- unzip -qq gradle-6.2.1-bin.zip
+- export GRADLE_HOME=$PWD/gradle-6.2.1
+- export PATH=$GRADLE_HOME/bin:$PATH
+- gradle -v
```

Comment:
I don't remember why installing Gradle wasn't necessary before (I'm mostly using GitLab since then), was it already part of the CI environment? If this is necessary, then using the wrapper might be better, it's why it's there. So IMO let's not gitignore it, add it with this version, and call gradlew in the CI scripts below. Any reason against it?

Reply:
At the time of implementing Gradle 6 was not shipped with Travis CI. I will check if it is now.